geode-pulse/build.gradle

```diff
@@ -169,6 +170,8 @@ artifacts {
 }
 
 war {
+  enabled = true
+  rootSpec.exclude("**/*commons-logging-*.jar")
```

Comment:
Out of curiosity: why are we excluding this library from the WAR file?.

Reply:
I was making the `war` spec the same as our other definitions. if they *should* be different, I'm happy to drop the modification here. The problem (and reason this is a draft PR) is that nobody has definitive answers to the expectations of these artifacts :(

Reply:
Fair enough. I don't have any strong opinions either, I'm just worried about breaking PULSE logging due to the removal of this particular dependency... maybe we can test that before merging the changes?.